### ISSUE TITLE
Add get_model method to Classifer 

### ISSUE THREAD

**Issue by davidslater** [IBM/adversarial-robustness-toolbox#371]
**Is your feature request related to a problem? Please describe.**
Adaptive attacks often require modification of the underlying model, such as by removing the softmax output or replacing operations in the network (e.g. replacing log(exp(x)) with x). However, it is difficult to access the underlying model after wrapping it in Classifier.

**Describe the solution you'd like**
A `get_model()` method in Classifier that returns the original underlying model.

**Describe alternatives you've considered**
I have used `classifier._model` to get access to the model and modify it for TF models.  However, the PyTorch wrapper adds an additional layer and `classifier._model._model` is needed.

**Comment by davidslater** [comment]
This would also make it easier to save models after adversarially training them with ART.
